PubSub/src/Subscription.php

```diff
@@ -202,6 +204,15 @@ public function name()
      *           messages, and thus configures how far back in time a `Seek`
      *           can be done. Cannot be more than 7 days or less than 10 minutes.
      *           **Defaults to** 7 days.
+     *     @type array $expirationPolicy A policy that specifies the conditions
+     *           for resource expiration (i.e., automatic resource deletion).
+     *     @type Duration $expiration.ttl Specifies the "time-to-live" duration
```

Comment:
Is it intended for this to accept strings as well?

Reply:
it will accept a string, but Duration is "more correct".

Reply:
Then should we document `Duration|string`?